### ISSUE TITLE
support for geyser

### ISSUE THREAD

**Issue by Creeperz653** [unnamed/hephaestus-engine#8]
once geyser finishes its API for custom models you should support bedrock entities

**Comment by Estrobeda** [comment]
Geyser seems to be compatible with custom models if they use armorstands assuming you manually provide the bedrock equivalent resource pack.

Thus this issue "should" be possible to achieve by making a bedrock resource pack generator as well. The tricky part, besides making said generator would be to move the resource pack into the geyser folder. 

Personally, I'd prefer bedrock's native custom entity system since that would be more optimized for the bedrock client but it seems like geysermc does not support custom entities through resource & datapacks sadly. At least not yet.

Model engine 3 seems to be working with geyser with at least some models while Model engine 4 is completely incompatible due to it's use of display entities which bedrock does not have.

**Comment by Creeperz653** [comment]
> Geyser seems to be compatible with custom models if they use armorstands assuming you manually provide the bedrock equivalent resource pack.
> 
> 
> 
> Thus this issue "should" be possible to achieve by making a bedrock resource pack generator as well. The tricky part, besides making said generator would be to move the resource pack into the geyser folder. 
> 
> 
> 
> Personally, I'd prefer bedrock's native custom entity system since that would be more optimized for the bedrock client but it seems like geysermc does not support custom entities through resource & datapacks sadly. At least not yet.
> 
> 
> 
> Model engine 3 seems to be working with geyser with at least some models while Model engine 4 is completely incompatible due to it's use of display entities which bedrock does not have.

Yes but if this api used geyser api it would probably save so much time then having to maunally implement it but the geyser native entitys will only progress if people commit and help
